src/server/api/routes/favorite.vessels.router.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
```

Comment:
Rename file to `favorite-vessels.router.js`

Reply:
Done ;)